### PR TITLE
Mini Rubric: Expand the selected performance level for student by default

### PR DESCRIPTION
**Pull request by dmcavoy**
Expand the performance level the teacher gave to a student (if they gave them rubric feedback) by default when the student is looking at feedback.

# Before
Screen Shot 2019-04-12 at 11 48 48 AM

# After
Screen Shot 2019-04-12 at 11 35 07 AM

### CODE REVIEW

File: apps/src/templates/instructions/TeacherFeedback.jsx [hunk 1]
```diff
@@ -263,7 +269,11 @@ export class TeacherFeedback extends Component {
                   <RubricField
                     key={level}
                     showFeedbackInputAreas={showFeedbackInputAreas}
-                    expandByDefault={this.props.displayKeyConcept}
+                    expandByDefault={
```
Comment (maddiedierker) [inline apps/src/templates/instructions/TeacherFeedback.jsx:272]: this is calculation is getting pretty complex, so let's assign its value to a variable at the top of the `render()` method. something like:
```
const expandByDefault = this.props.displayKeyConcept || (expandPerformanceLevelForStudent && this.state.performance === level)
```

Comment (maddiedierker) [reply apps/src/templates/instructions/TeacherFeedback.jsx:272]: similar to what you're doing for `expandPerformanceLevelForStudent`

**Review by maddiedierker — APPROVED** [review]
i left a minor inline comment, but this looks great!

File: apps/src/templates/instructions/TeacherFeedback.jsx (hunk 1 above)
Comment (dmcavoy) [reply apps/src/templates/instructions/TeacherFeedback.jsx:272]: The level here comes from the map which is on the line right above <RubricField. In order to move this out I think I would need to make it a function. Should I do that?

Comment (dmcavoy) [reply apps/src/templates/instructions/TeacherFeedback.jsx:272]: The level here comes from the map which is on the line right above <RubricField. In order to move this out I think I would need to make it a function. Should I do that?

Comment (maddiedierker) [reply apps/src/templates/instructions/TeacherFeedback.jsx:272]: ah, sorry, i should've taken a closer look at the context here. in that case, it's fine to leave as-is

**Pull request merged**